lms/djangoapps/instructor_task/tasks.py

```diff
@@ -155,6 +156,24 @@ def calculate_grades_csv(entry_id, xmodule_instance_args):
     return run_main_task(entry_id, task_fn, action_name)
 
 
+# TODO: GRADES_DOWNLOAD_ROUTING_KEY is the high mem queue.  Do we know we need it?
```

Comment:
Please use the high mem queue. Grading can run a very long time, and monitoring of it has shown that the memory used only grows over time after successive runs. The main difference between queues is that high mem will force the worker to restart after every job. Given the usage pattern for grading, this should be fine.